.circleci/test.yml

```diff
@@ -1,31 +1,38 @@
 version: 2.1
 
 orbs:
-  agentic-cli: cci-labs/agentic-cli@0.0.2
+  agentic-cli: cci-labs/agentic-cli@0.0.9
 
 jobs:
-  install-ai-clis:
+  test-environment-variables:
     docker:
       - image: cimg/base:current
     environment:
       WEBHOOK_BODY: << pipeline.trigger_parameters.webhook.body >>
       TRIGGER_SOURCE: << pipeline.trigger_source >>
-      EVENT_TYPE: << pipeline.trigger_parameters.circleci.event_type >>
-      EVENT_NAME: << pipeline.event.name >>
-      EVENT_ACTION: << pipeline.event.action >>
     steps:
       - checkout      
-      # Print versions of all installed CLIs
       - run:
-          name: "Print Certain Environment Variables"
+          name: "Print Environment Variables"
           command: |
+            echo "=== Environment Variables Debug ==="
             echo "WEBHOOK_BODY: $WEBHOOK_BODY"
             echo "TRIGGER_SOURCE: $TRIGGER_SOURCE"
-            echo "EVENT_TYPE: $EVENT_TYPE"
-            echo "EVENT_NAME: $EVENT_NAME"
-            echo "EVENT_ACTION: $EVENT_ACTION"
+            
+            # Safely check for optional variables
+            if [ -n "${CIRCLE_PIPELINE_ID:-}" ]; then
+              echo "CIRCLE_PIPELINE_ID: $CIRCLE_PIPELINE_ID"
```

Comment:
🛡️ **Excellent defensive programming!** Using `${CIRCLE_PIPELINE_ID:-}` pattern prevents shell errors if variables are undefined. This is a best practice for handling optional CircleCI environment variables that may not always be available.